client/src/css/background-animation.css

```diff
@@ -0,0 +1,135 @@
+@-webkit-keyframes gradientAnimation {
+    0%{background-position:0% 50%}
+    50%{background-position:100% 50%}
+    100%{background-position:0% 50%}
+}
+@-moz-keyframes gradientAnimation {
```

Comment:
i don't know about correct codestyle for front, but maybe we should make a single rule for the number of blank lines between blocks (i mean, in some places there are no empty lines at all, somewhere one, somewhere two)